omf-website/src/components/code-snippet-v2.tsx

```
@@ -0,0 +1,484 @@
+                <rect width="14" height="14" x="8" y="8" rx="2" ry="2" />
+                <path d="M4 16c-1.1 0-2-.9-2-2V4c0-1.1.9-2 2-2h10c1.1 0 2 .9 2 2" />
+              </svg>
+              Copy
+            </span>
+          )}
+        </Button>
+      </div>
+      <pre className="p-4 overflow-x-auto bg-[#1e293b] rounded-b-lg">
+        <code
+          className="text-sm font-mono text-gray-200 leading-relaxed"
+          dangerouslySetInnerHTML={{ __html: highlightCode(code, language) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)